Sources/Valet/Internal/SecItem.swift

```diff
@@ -70,23 +70,34 @@ internal final class SecItem {
 
     /// Programatically grab the required prefix for the shared access group (i.e. Bundle Seed ID). The value for the kSecAttrAccessGroup key in queries for data that is shared between apps must be of the format bundleSeedID.sharedAccessGroup. For more information on the Bundle Seed ID, see https://developer.apple.com/library/ios/qa/qa1713/_index.html
     internal static var sharedAccessGroupPrefix: String {
-        let query = [
+        var query: [CFString : Any] = [
             kSecClass : kSecClassGenericPassword,
             kSecAttrAccount : "SharedAccessGroupAlwaysAccessiblePrefixPlaceholder",
             kSecReturnAttributes : true,
-            kSecAttrAccessible : Accessibility.alwaysThisDeviceOnly.secAccessibilityAttribute
-            ] as CFDictionary
-        
+            kSecAttrAccessible : Accessibility.alwaysThisDeviceOnly.secAccessibilityAttribute,
+        ]
+
+        if #available(iOS 13.0, tvOS 13.0, watchOS 6.0, macOS 10.15, *) {
+            // Update the account to be distinct, since we're changing the query.
+            query[kSecAttrAccount] = "SharedAccessGroupAlwaysAccessiblePrefixPlaceholderWithDataProtection"
```

Comment:
If we don't do this, a `SharedAccessGroupAlwaysAccessiblePrefixPlaceholder` that was added without `kSecUseDataProtectionKeychain` wouldn't be found on `SecItemCopy`, but trying to add that account/data to the keychain would return a `errSecDuplicateItem`.

Reply:
Nevermind. Looks like `errSecDuplicateItem` is only returned when a `kSecAttrService` or `kSecAttrAccount` overlaps. Removing this line.